bundles/CoreBundle/Migrations/Version20210211152319.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace Pimcore\Bundle\CoreBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Pimcore\Migrations\Migration\AbstractPimcoreMigration;
+use Pimcore\Model\DataObject\ClassDefinition;
+use Pimcore\Model\DataObject\Objectbrick\Definition;
+
+class Version20210211152319 extends AbstractPimcoreMigration
+{
+    public function doesSqlMigrations(): bool
+    {
+        return false;
+    }
+
+    /**
+     * @param Schema $schema
+     *
+     * @throws \Exception
+     */
+    public function up(Schema $schema)
+    {
+        $list = new ClassDefinition\Listing();
+        $list = $list->load();
+        foreach ($list as $class) {
+            $this->writeMessage(sprintf('Saving php files for class: %s', $class->getName()));
+            $class->generateClassFiles(false);
```

Comment:
generating classes is not enough.....we need to up db tables as well for column type update

Reply:
Even calling `$class->save()` in migration has no impact on existing classes.